### PR TITLE
Changelog for 1.7.6.5

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Changelog for 1.7.6.5
| Type?         | improvement
| Category?     | CO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18659)

### CODE REVIEW

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -24,6 +24,32 @@ International Registered Trademark & Property of PrestaShop SA
...
+    - #18064: Fix error when trying to translate Serbian using the BO interface (by @eternoendless)
+- Front Office:
+  - Bug fix:
+    - #18633: Convert cart rule value when order currency is different (by @sowbiba)
+    - #18493: Change product redirection rules to redirect to valid attribute url (by @jolelievre)
+    - #18103: Duplicate address when submitting a form with errors (by @PierreRambaud)
+- Core:
+  - Improvement:
+    - #18638: Update version to 1.7.6.5 (by @PierreRambaud)
+    - #18013: Add PululuK as a contributor (by @LouiseBonnard)
```
Comment (eternoendless) [inline docs/CHANGELOG.txt]: This line can be deleted I think

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
We need to update dependencies first

**Review by Progi1984 — DISMISSED** [review]

File: docs/CHANGELOG.txt [hunk 2]
```diff
@@ -24,6 +24,47 @@ International Registered Trademark & Property of PrestaShop SA
...
+    - #18064: Fix error when trying to translate Serbian using the BO interface (by @eternoendless)
+- Front Office:
+  - Bug fix:
+    - #18633: Convert cart rule value when order currency is different (by @sowbiba)
+    - #18493: Change product redirection rules to redirect to valid attribute url (by @jolelievre)
+    - #18103: Duplicate address when submitting a form with errors (by @PierreRambaud)
+- Core:
+  - Improvement:
+    - #18638: Update version to 1.7.6.5 (by @PierreRambaud)
+  - Security fix:
```
Comment (sowbiba) [inline docs/CHANGELOG.txt]: I will add the CVE in front of each fix

**Review by jolelievre — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @sowbiba

**Pull request merged**